src/generic_ui/polymer/cloud-install.html

```diff
@@ -32,20 +29,26 @@
     }
     uproxy-button {
       background-color: #34AFCE;  /* blue */
+      margin: 4px 0;
       padding: 0.5em;
       width: 100%;
       font-size: 14px;
+      text-transform: none;
```

Comment:
IIRC, I stole these styles from other buttons in the system - now we will have a mix of `uppercase` and regular lettering, is that intentional?